.idea/.idea.osu/.idea/indexLayout.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
-  <component name="ContentModelUserStore">
+  <component name="UserContentModel">
```

Comment:
This suggests you're not using the `osu.Desktop.slnf` project, which you probably should be (it doesn't contain mobile projects and therefore is quite a lot faster to work with).

Probably okay to just let this one pass as I believe this is the new correct value, just hasn't been applied yet because no one uses the main `sln` file.

Reply:
Ah sorry, I didn't mean to commit that. You're right, I was using `osu.sln` but had all the non-desktop stuff unloaded. Rider also automatically changed `.idea/.idea.osu/.idea/projectSettingsUpdater.xml` and deleted `.idea/.idea.osu/.idea/modules.xml`.

I'll use `osu.Desktop.slnf` in the future.